mpt/impl/smtp/cassandra/src/test/java/org/apache/james/mpt/smtp/host/CassandraJamesSmtpHostSystem.java

```diff
@@ -101,10 +104,16 @@ public InMemoryDNSService getInMemoryDnsService() {
         return inMemoryDNSService;
     }
 
-    protected GuiceJamesServer createJamesServer() {
-        return new GuiceJamesServer()
+    protected GuiceJamesServer createJamesServer() throws IOException {
+        String workingDir = folder.newFolder().getAbsolutePath();
```

Comment:
I don't get why you call newFolder(). As folder is already a temporary one, you just need folder.getFile().getAbsolutePath(), no? (and everywhere in this PR, I fixed only one of them to make the test pass for the moment)

Reply:
A TemporaryFolder is not a folder by itself hence these calls.

Reply:
Are you sure? In my test cases it resolves to /tmp/junitXXX

Reply:
Well, uncompiling JUNIT 4 : 

```
public class TemporaryFolder extends ExternalResource {
    private final File parentFolder;
    private File folder;
// ...
```

You can see that : 
 - This class is not a File but a wrapper around a file.
 - Some kind of unwrapping is needed, like `getRoot()` or `newFolder()`

Reply:
yes, that's why I suggested folder.getFile() instead of folder.newFolder().

Reply:
I'm not sure because doing that you may provide different working directory for different part of james, while they assume to have a single directory. See my fixup https://github.com/linagora/james-project/pull/1389/commits/2ab86bc417540a56255ebf4f10cd137b35ffafae (ooh it seems I used getRoot() by the way)